### PR TITLE
Enrich pull request template

### PR DESCRIPTION
**Pull request by Andreagit97**
**What type of PR is this?**

/kind design

/kind documentation

**Any specific area of the project related to this PR?**

**What this PR does / why we need it**:

This PR tries to enrich the pull request template with some new labels

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by jasondellaluce — APPROVED** [review]
/approve

Thanks, this was kind of outdated!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -75,5 +79,5 @@ For example, `action required: change the API interface of the rule engine`.
 -->
 
 ```release-note
-
+NONE
```
Comment (leogr) [inline .github/PULL_REQUEST_TEMPLATE.md:82]: I'd not set `NONE` by default, otherwise, folks will just forget to add a release note.

**Review by leogr — COMMENTED** [review]
Just a comment, otherwise SGTM :+1:

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Andreagit97) [reply .github/PULL_REQUEST_TEMPLATE.md]: fixed :)

**Review by jasondellaluce — APPROVED** [review]
/approve

**Review by leogr — APPROVED** [review]

**Pull request merged**